src/js/control-bar/play-toggle.js

```diff
@@ -101,6 +99,7 @@ class PlayToggle extends Button {
    * @listens Player#pause
    */
   handlePause(event) {
+    this.removeClass('vjs-ended');
```

Comment:
not sure we want this. If you're ended and the player is told to pause, it should look the same.

Reply:
agreed, this ended thing is starting to look messy (see extra releases just to fix things related to this). +1 for not making the play-toggle more complex (which trickles down to plugins).